<launch>
    <param name="map_path" type="string" value="/home/gi/Documents/LeGO_LOAM_ENV/obstacle_avoidance_rover_legoloam_map_pcd/pcd/map_global_registered.pcd"/>
    <param name="dynamic_block_generator_downsampling_size" type="double" value="0.4"/>
    <param name="dynamic_block_obstacle_range" type="double" value="30"/>
    <node name="dynamic_map_block_generator_node" pkg="obstacle_and_drivable_block_publisher" type="dynamic_block_generator" output="screen"/>
</launch>
